src/test/java/io/vertx/it/MSSQLDecoder.java

```diff
@@ -0,0 +1,30 @@
+package io.vertx.it;
```

Comment:
For Eclipse Compliance, can you add a copyright header just like the other files?